server/server.go

```diff
@@ -51,6 +52,15 @@ type config struct {
 	KeployApp        string `envconfig:"APP_NAME" default:"Keploy-Test-App"`
 }
 
+//returns the env value if any else default fallback
+func getenv(key, fallback string) string {
```

Comment:
@Pradhyuman-sharma we already have env variable support via config libraries. please use that

Reply:
Done 👍